WireAPI/Sources/WireAPI/APIs/UpdateEventsAPI/Event decoding/UpdateEvent+Decodable.swift

```diff
@@ -26,22 +26,22 @@ extension UpdateEvent: Decodable {
 
         do {
             switch UpdateEventType(eventType) {
-            case .conversation(let eventType):
+            case let .conversation(eventType):
```

Comment:
This rule doesn't fit my preferences :/ should we discuss rules as team and decide?